### DISCUSSION TITLE
Onprem/local LLM's

### DISCUSSION THREAD

**Discussion by rgidda** [eidolon-ai/eidolon#959] [General]
Does Eidolon-AI support the use of self-hosted local LLMs? This would provide users with the flexibility to keep their data within their organization, avoiding external transmission.

**Comment by LukeLalor** [comment]
Hi, sorry about the late response. We haven't yet set up automatic tagging on discussions.

Eidolon does support local plan via ollama. In our documentation you can read how to configure the ollama llm provider.
